daml-lf/kvutils/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_library",
+    "da_scala_test_suite",
+    "lf_scalacopts",
+)
+load("//bazel_tools:scala.bzl", "silencer_plugin")
+
+da_scala_library(
+    name = "kvutils",
```

Comment:
Do we need to publish the jar on maven ? 
Is there is a way to ship in the jar `//ledger/participant-state/kvutils:kvutils` ?

Reply:
not in any nice way. If it’s a separate bazel target it’s going to be published separately.

Reply:
I add the maven tag just below and the entry in `release/artifacts.yaml`. Do I miss something ?

Reply:
that should be sufficient. In general, CI should yell at you if you forget an entry.

Reply:
:+1: